CGFloatType/CGFloatType.h

```diff
@@ -37,6 +37,9 @@ CGFloat CGFloatWithDouble(double value);
 /// Returns a CGFloat from a float
 CGFloat CGFloatWithFloat(float value);
 
+#pragma mark - CGFloat comparison
+
+extern BOOL CGFloatEquals(CGFloat x, CGFloat y);
```

Comment:
I don't know, almost every function uses `extern` so I kept it. I can remove it if you think it's better